src/app/dashboard-widgets/pipelines-widget/pipelines-widget.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="pipelines-widget card-pf">
+  <div class="flex-container in-column-direction">
+    <h2 class="card-pf-title card-pf-utilization-details">
+      <span>Pipelines</span>
+      <span class="badge">{{buildConfigsCount | async}}</span>
+    </h2>
+    <div class="card-pf-body">
+      <div class="blank-slate-pf" *ngIf="(buildConfigsCount | async) === 0">
+        <div class="blank-slate-pf-icon">
+          <span class="fa fa-code-fork"></span>
+        </div>
+        <h1>There are no pipelines.</h1>
+        <p>
+          Pipelines will appear when you commit your code.
+        </p>
+      </div>
+      <div class='list-view-pf-main-info' *ngIf="(buildConfigsCount | async) > 0">
+        <div class='list-view-pf-left'>
+          <p *ngFor="let buildconfig of buildConfigs | async">
+            <span class="{{buildconfig.iconStyle}}" title="{{buildconfig.statusPhase}}"></span> Last builld for <em>{{ buildconfig.name
+            }}</em> was <em>#{{buildconfig.lastVersion}}</em>, ran
```

Comment:
can you clean up this html, the line break makes it harder to read and it could use another layer of indention